### ISSUE TITLE
Banners in Global Settings: Painting icon doesn't support keyboard navigation

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#13426]
**Setup**
- Rancher version: 
Rancher | v2.11-37b99eef43096c288c498f431155c331cd9eaaa7-head
Dashboard | master 2594e5e
Rancher CLI | v2.11.0-rc.1

**Describe the bug**
In Header, Footer and Login Screen banners, the painting icon of both Text Color and Background Color is accessible via keyboard navigation. However, the feature is not supported, as the user is not able to move the painting icon along the screen with arrow keys.

**To Reproduce**
1- Under the burguer menu, select Global Settings
2- From the Global settings side menu, select Banners
3- Enable Show Banner
4- Using the keyboard navigation, select the painting icon to change the text/background color 
5- Try to navigate on the screen to change the text/background color using the painting icon feature

**Result**
User is not able to move the painting icon along the screen with arrow keys.

**Expected Result**
User should be able to move the painting icon along the screen with arrow keys to select a color using the painting icon.

**Screenshots**

![Image](https://github.com/user-attachments/assets/5ee19ff7-f934-44b5-b0f0-b2ef69481487)

**Comment by aalves08** [comment]
@rak-phillip moving this to `2.12` milestone with `priority 2`. I don't think we can do anything about it because it's a native browser input, nevertheless we can do a bit of digging around once we are out of `a11y` priority work.

Feel free to change if you don't agree mate 🙏

**Comment by aalves08** [comment]
@IsaSih i've researched about this and I couldn't find any solution nor any document that points to the fact that we can't address this.

This input is a "native" HTML solution https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color and it's usability is also tied to the browser support, which is pretty good given it's "exotic" nature (not with really common usage, for sure). We can't fix this.

Given this fact, plus the fact that all other different interaction elements in this input allow for a complete keyboard navigation I think that is enough to close this issue as we can't do anything about the eyedropper.

FYI @nwmac @kwwii
